packages/meteor/deprecate.js

```diff
@@ -1,3 +1,26 @@
+if (Meteor.isServer && Meteor.isDevelopment) {
+  if (typeof __meteor_runtime_config__ === 'object') {
+    var noDeprecation = process.env.METEOR_NO_DEPRECATION || process.noDeprecation;
+    if (noDeprecation === 'true' || noDeprecation === 'false') {
+      noDeprecation = noDeprecation === 'true';
+    }
+    __meteor_runtime_config__.noDeprecation = noDeprecation;
+  }
+}
+
+function oncePerArgument(func) {
+  var cache = new Map();
```

Comment:
Is there any chance of the cache growing indefinitely if the arguments change, causing a memory leak or hitting 16m items?

Reply:
We don't use that amount of Meteor.deprecate I think hehe